randovania/layout/description_migration.py

```diff
@@ -148,14 +148,15 @@ def _migrate_v6(json_dict: dict) -> dict:
     }
 
     for game in json_dict["game_modifications"]:
-        for area_name, identify_game in area_name_heuristic.items():
+        for (i, (area_name, identify_game)) in enumerate(area_name_heuristic.items()):
             if area_name in game["locations"]:
+                if identify_game == "prime1":
+                    json_dict["game_modifications"][i]["locations"]["Frigate Orpheon"] = dict()
```

Comment:
just alter `game` directly, it's the same as `json_dict["game_modifications"][i]`.